test/jaas-test/src/main/java/com/griddynamics/jagger/test/jaas/invoker/InvokerWithoutStatusCodeValidation.java

```diff
@@ -7,6 +7,10 @@
 import java.util.HashMap;
 import java.util.Map;
 
+/**
+ * Invoker without checking result codes.
+ * I.e. if request returns 4xx or 5xx this invoker doesn't throw exception unlike DefaultHttpInvoker
+ */
```

Comment:
such behavior of the Default invoker is a bug
https://issues.griddynamics.net/browse/JFG-1059